diff --git a/ROS/src/sim_robot/launch/spring_experiment.launch.py b/ROS/src/sim_robot/launch/spring_experiment.launch.py
index 4b84120..ef768d4 100644
--- a/ROS/src/sim_robot/launch/spring_experiment.launch.py
+++ b/ROS/src/sim_robot/launch/spring_experiment.launch.py
@@ -77,7 +77,7 @@ def launch_setup(context, *args, **kwargs):
         parameters=[{
             'config_file': os.path.join(pkg_share, 'config',
                                         'ros_gz_bridge_spring.yaml'),
-            'expand_gz_topic_names': True,
+            # 'expand_gz_topic_names': True,
         }],
         output='screen',
     )
